src/common/config.rs

```diff
@@ -80,62 +65,72 @@ fn validate_ed25519_pubkeys(pubkeys: Vec<&HexEd25519Key>) -> bool {
 }
 
 fn validate_ed25519_privkey(keypair: &HexPubPriv) -> bool {
-        let privkey_provided_bytes: Vec<u8> = match keypair.privkey.to_owned().try_into() {
-            Ok(bytes) => bytes,
-            _ => return false
-        };
-        match SigningKey::try_from(privkey_provided_bytes.as_slice()) {
-            Ok(privkey_provided) => {
-                let pubkey_provided_bytes: Vec<u8> = match keypair.pubkey.to_owned().try_into() {
-                    Ok(bytes) => bytes,
-                    _ => return false
-                };
-                let pubkey_computed = VerificationKeyBytes::from(&privkey_provided);
-                pubkey_provided_bytes == pubkey_computed.as_ref()
-            },
-            _ => false
+    let privkey_provided_bytes: Vec<u8> = match keypair.privkey.to_owned().try_into() {
+        Ok(bytes) => bytes,
+        _ => return false,
+    };
+    match SigningKey::try_from(privkey_provided_bytes.as_slice()) {
+        Ok(privkey_provided) => {
+            let pubkey_provided_bytes: Vec<u8> = match keypair.pubkey.to_owned().try_into() {
+                Ok(bytes) => bytes,
+                _ => return false,
+            };
+            let pubkey_computed = VerificationKeyBytes::from(&privkey_provided);
+            pubkey_provided_bytes == pubkey_computed.as_ref()
         }
```

Comment:
```suggestion
        },
```
to be consistent with your previous style adjustments.

Reply:
I just used `rustfmt`

Reply:
Ah ok, then disregard my suggestion